test/ui-test/testSuites/global_shared/steps/steps.py

```diff
@@ -88,9 +88,9 @@ def step(context, obj):
 ###########################################################################
 
 def the_user_restarts_the_app(context: any):
-    waitFor(lambda: currentApplicationContext().detach(), 500)
+    StartupSteps.waitFor(lambda: StartupSteps.currentApplicationContext().detach(), 500)
```

Comment:
I think neither `waitFor` not `currentApplicationContext()` are part of `StartupSteps`. Just discard this changes!